<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Analyzer</class>
 <widget class="QDialog" name="Analyzer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QListWidget" name="video_part_list">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>52</y>
     <width>1020</width>
     <height>668</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
  </widget>
  <widget class="QToolButton" name="max_video">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>10</y>
     <width>80</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>1080P</string>
   </property>
   <property name="popupMode">
    <enum>QToolButton::MenuButtonPopup</enum>
   </property>
  </widget>
  <widget class="QLabel" name="vq_label">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>10</y>
     <width>60</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>最高画质：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="aq_text">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>10</y>
     <width>60</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>最高音质：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QToolButton" name="max_audio">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>10</y>
     <width>80</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>132K</string>
   </property>
   <property name="popupMode">
    <enum>QToolButton::MenuButtonPopup</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="apply_filter_btn">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>10</y>
     <width>84</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>应用到全部</string>
   </property>
  </widget>
  <widget class="QPushButton" name="download_cancle">
   <property name="geometry">
    <rect>
     <x>938</x>
     <y>730</y>
     <width>84</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="select_all_btn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>730</y>
     <width>84</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>全选</string>
   </property>
  </widget>
  <widget class="QPushButton" name="select_reverse_btn">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>730</y>
     <width>84</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>反选</string>
   </property>
  </widget>
  <widget class="QPushButton" name="select_no_btn">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>730</y>
     <width>84</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>全不选</string>
   </property>
  </widget>
  <widget class="QToolButton" name="download_btn">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>730</y>
     <width>84</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>下载选中项</string>
   </property>
   <property name="popupMode">
    <enum>QToolButton::MenuButtonPopup</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="path_line">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>730</y>
     <width>411</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolButton" name="path_btn">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>730</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="info_fetch_bar">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>720</y>
     <width>1020</width>
     <height>3</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QProgressBar{text-align:center;background-color:#dfe6e9;}
QProgressBar::chunk{background-color:#fd79a8;}</string>
   </property>
   <property name="maximum">
    <number>0</number>
   </property>
   <property name="value">
    <number>-1</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="fmt_select">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>10</y>
     <width>80</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>MP4</string>
   </property>
   <property name="popupMode">
    <enum>QToolButton::MenuButtonPopup</enum>
   </property>
  </widget>
  <widget class="QLabel" name="fmt_text">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>60</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>输出格式：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QToolButton" name="url_btn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>221</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>113696558814120</string>
   </property>
   <property name="popupMode">
    <enum>QToolButton::MenuButtonPopup</enum>
   </property>
  </widget>
  <widget class="QProgressBar" name="analyzer_init_bar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>41</y>
     <width>216</width>
     <height>3</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QProgressBar{text-align:center;background-color:#dfe6e9;}
QProgressBar::chunk{background-color:#fd79a8;}</string>
   </property>
   <property name="maximum">
    <number>0</number>
   </property>
   <property name="value">
    <number>-1</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="codec_text">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>10</y>
     <width>60</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>输入编码：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QToolButton" name="codec_select">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>10</y>
     <width>80</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>AVC</string>
   </property>
   <property name="popupMode">
    <enum>QToolButton::MenuButtonPopup</enum>
   </property>
  </widget>
  <widget class="QLabel" name="current_select_count_label">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>730</y>
     <width>108</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>current: 0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="total_count_label">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>total: 0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="fetch_count_label">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>24</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>fetch: 0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="total_select_count_label">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>744</y>
     <width>108</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>total: 0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
